### PR TITLE
relay card and relay page + minor corrections

### PR DESCRIPTION
**Pull request by 123vier**

### CODE REVIEW

**Review by KoalaSat — APPROVED** [review]
Danke schön!

**Comment by 123vier** [comment]
> 

You're welcome. Happy to contribute :)

File: frontend/Locales/de.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+      "relaysDescription": "Relays sind Nodes (Netzknoten) im Netzwerk, die als Vermittler von Nachrichten zwischen den Anwendungen dienen.\n\n\nRelays können die Belastbarkeit und die Verfügbarkeit des Netzwerks verbessern, indem sie dafür sorgen, das Nachrichten trotz Unterbrechungen oder Ausfällen der Verfügbarkeit ausgeliefert werden.\n\n\nRelays können Privatsphäre und Netzwerksicherheit erhöhen, indem sie Aufenthaltsort und Identität der Anwendungen bzw. der Benutzer, die miteinander kommunizieren, verbergen\n\n\nDies kann von Wert sein in Umgebungen, in denen Zensur oder Überwachung ein Problem ist.\n\n\nEs ist wichitg darauf hinzuweisen, das Relays für schadhafte Zwecke missbraucht werden können, wie zum Beispiel Sniffing oder das Zensieren von Netzwerkverkehr.\n\n\nDeswegen ist es von Bedeutung, die Nutzung von Relays sorgfältig abzuwägen, und angemessenene Sichheitsmassnahmen anzuwenden, um Identität, Privatsphäre und Netzwerksicherheit zu schützen.",
```
Comment (kaiwolfram) [inline frontend/Locales/de.json:5]: ~~wichitg~~ wichtig

Comment (123vier) [reply frontend/Locales/de.json:5]: Vielen Dank, wird korrigiert :)

Comment (KoalaSat) [reply frontend/Locales/de.json:5]: changed on an unrelated branch I'm working on, will be merge together, thanks!

**Pull request merged**